internal/scorecard/alpha/tests/olm.go

```diff
@@ -83,3 +107,77 @@ func StatusDescriptorsTest(bundle registry.Bundle) scapiv1alpha2.ScorecardTestRe
 	r.Suggestions = make([]string, 0)
 	return r
 }
+
+func validateCR(cr unstructured.Unstructured, crds []*v1beta1.CustomResourceDefinition,
+	r scapiv1alpha2.ScorecardTestResult) scapiv1alpha2.ScorecardTestResult {
+
+	// check if the CRD matches the testing CR
+	for _, crd := range crds {
+		gvk := cr.GroupVersionKind()
+		// Only check the validation block if the CRD and CR have the same Kind and Version
+		if !(matchVersion(gvk.Version, crd) && matchKind(gvk.Kind, crd.Spec.Names.Kind)) {
+			continue
+		}
+		if crd.Spec.Validation == nil {
+			r.Suggestions = append(r.Suggestions, fmt.Sprintf("Add CRD validation for %s/%s",
+				crd.Spec.Names.Kind, crd.Spec.Version))
+			continue
```

Comment:
should it go to the next checks when the spec.Validation is nill? Should not the other checks also made?

Reply:
@jmccormick2001 @fabianvf any thoughts here? I do not want to alter the logic without knowing its purpose at this point. If you guys have anything to add regarding the reason, Please share.

Reply:
it looks like the rest of the checks rely on the `crd.Spec.Validation` being present, removing the `continue` here would just make it so that you had to do additional checks later on to skip the validations anyway

Reply:
Yep, it is true. I re-checked now. If has not Validation spec has no longer checks to be done. 
You can ignore my comment @bharathi-tenneti